src/compiler/utilities.ts

```diff
@@ -5953,6 +5953,10 @@ namespace ts {
         return isOptionalChainRoot(node.parent) && node.parent.expression === node;
     }
 
+    export function isQuestionQuestionExpression(node: Node) {

```

Comment:
The analogous factory function is `createNullishCoalesce`, so I think this should be `isNullishCoalesce`.

Reply:
Yeah, I guess, though for the uninitiated, nullish coalesce is meaningless and question question is self-explanatory. But hey, gotta have something for the quizzes. 😏

Reply:
How else do you expect me to rig TSConf trivia?